src/class-actions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * The Filters class, used for defining available filters
+ *
+ * @package Eightshift_Forms\Core
+ */
+
+declare( strict_types=1 );
+
+namespace Eightshift_Forms\Core;
+
+/**
+ * The Actions class, used for defining available actions.
+ */
+class Actions {
+
+  /**
+   * This action should be used when doing Buckaroo integration. It will run after the user
+   * has been redirected from Buckaroo (with appropriate status - success, cancel, error, reject)
+   * back to our site but before he is redirected to the success / error page defined in Form's options.
+   *
+   * The action's callback needs to be provided in your project.
+   *
+   * @var string
+   */
+  const BUCKAROO_RESPONSE_HANDLER = 'eightshift_forms/buckaroo_response_handler';
```

Comment:
So the only usage of this class is to define the constant?  👀 

You could have used interface for it, but still, I'm failing to see the reasoning behind extracting this to a class. Unless it's reused in 10 different places, in which case interface makes a bit more sense (to me).

Reply:
There is another class for `filters` (`class-filters.php`) which has ~9 filters. This one is for `actions`. Coincidentally there's only 1 action right now but this will probably change in the future.

It serves as a simple list of available filters / actions - both for future documentation as well as just you know, to keep all these constants in a single place. I could have put it in `class-filters.php` but I wanted to keep things separated and more "correct". 

Also I kinda fail to see how an interface would solve this. The point of both of these classes is to be able to reference those filter / action names in eightshift-forms without having to write them as strings each time (reusing code).

Reply:
To me, these look kinda like enums, even though their use case is a bit different. 

I just see classes as encapsulated code that holds logic. Constants are just names without any logic whatsoever. In that aspect, an interface makes more sense. No logic, just a contract and you have encapsulation. You can implement numerous different interfaces in different places where it makes sense. Like `RouteInterface` in the libs. It only holds the constants for the HTTP verbs.

Also, these constants should all have visibility modifiers next to them (missed that in the first round) 😄